e2e/e2e_suite_test.go

```diff
@@ -43,18 +42,6 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	tmpPath.SetPath(path.Dir(shell.CurrentDir())+"/dist", os.Getenv("PATH"))
 	ensureRequiredBinaries()
 
-	shouldManageCluster := manageCluster()
-
-	if shouldManageCluster {
-		tmpClusterDir = TmpDir(GinkgoT(), "/tmp/ike-e2e-tests/cluster-maistra-"+naming.RandName(16))
-		executeWithTimer(func() {
-			fmt.Printf("\nStarting up Openshift/Istio cluster in [%s]\n", tmpClusterDir)
-			projectDir := testshell.GetProjectDir()
-			Expect(os.Setenv("IKE_CLUSTER_DIR", tmpClusterDir)).ToNot(HaveOccurred())
-			<-testshell.ExecuteInDir(projectDir, "make", "start-cluster").Done()
-		})
-	}
-
 	executeWithTimer(func() {
 		LoginAsTestPowerUser()
 
```

Comment:
Also this needs cleanup:

https://github.com/Maistra/istio-workspace/blob/1a342b0a7f9dae4c9ceee19037d14c9091a13d25/e2e/e2e_suite_test.go#L62-L66

Reply:
90908bf